### PR TITLE
Make it compatible with Windows

### PR DESCRIPTION
**Pull request by tong1wu**
Add macros to make this plugin work on Windows.

Signed-off-by: Tong Wu <tong1.wu@intel.com>

### CODE REVIEW

File: gstchecksumsink.c [hunk 1]
```diff
@@ -258,7 +263,7 @@ open_raw_file (GstCksumImageSink * checksumsink)
     return TRUE;
   else if (checksumsink->raw_file_name)
     checksumsink->fd = g_open (checksumsink->raw_file_name, O_WRONLY | O_CREAT,
-        S_IRUSR | S_IWUSR | S_IRGRP | S_IWGRP | S_IROTH);
+        0);
```
[the remark this answers is not in the document]
Comment (tong1wu) [reply gstchecksumsink.c:269]: ok, will fix it

File: gstchecksumsink.c [hunk 2]
```diff
@@ -435,7 +449,7 @@ gst_cksum_image_sink_render (GstBaseSink * sink, GstBuffer * buffer)
     n_planes = GST_VIDEO_FRAME_N_PLANES (&frame);
 
     for (plane = 0; plane < n_planes; plane++) {
-      gpointer pd = GST_VIDEO_FRAME_PLANE_DATA (&frame, plane);
+      guint8 *pd = GST_VIDEO_FRAME_PLANE_DATA (&frame, plane);
```
Comment (mengker33) [inline gstchecksumsink.c]: frame->data is stated as type gpointer, maybe we don't need to change it? Is it better to use a forced cast if there is a reason to pass frame->data to a guint8 pointer?

Comment (tong1wu) [reply gstchecksumsink.c]: fixed, thanks.

**Comment by uartie** [comment]
LGTM

**Pull request merged**